packages/commonwealth/client/scripts/views/pages/notification_settings2/SubEntry.tsx

```diff
@@ -0,0 +1,125 @@
+import { ThreadSubscription } from '@hicommonwealth/schemas';
+import { getThreadUrl } from '@hicommonwealth/shared';
+import { notifySuccess } from 'controllers/app/notifications';
+import { getRelativeTimestamp } from 'helpers/dates';
+import React, { useCallback } from 'react';
+import { Link, useNavigate } from 'react-router-dom';
+import { getCommunityUrl } from 'utils';
+import { trpc } from 'utils/trpcClient';
+import { CWCommunityAvatar } from 'views/components/component_kit/cw_community_avatar';
+import { CWText } from 'views/components/component_kit/cw_text';
+import { CWThreadAction } from 'views/components/component_kit/new_designs/cw_thread_action';
+import { User } from 'views/components/user/user';
+import { z } from 'zod';
+
+interface SubscriptionEntryProps {
+  readonly thread: z.infer<typeof ThreadSubscription>['Thread'];
+  readonly onUnsubscribe: (id: number) => void;
+}
+
+export const SubEntry = (props: SubscriptionEntryProps) => {
+  const { thread, onUnsubscribe } = props;
+  const thread_id = thread.id;
+
+  const threadUrl = getThreadUrl(
+    {
+      chain: thread.community_id,
+      id: thread.id,
+      title: thread.title,
+    },
+    undefined,
+    true,
+  );
+
+  const navigate = useNavigate();
+
+  const handleComment = useCallback(() => {
+    navigate(threadUrl);
+  }, [navigate, threadUrl]);
+  const deleteThreadSubscriptionMutation =
+    trpc.subscription.deleteThreadSubscription.useMutation();
```

Comment:
okay so for couple last months we were moving from old approach of fetching data to react-query. We decided that all our "interfaces" will sit in `state/api/*` directories. I would love to stick to this regardless we use trpc, reqct-query or whatever. The reason for this is that we don't do the mess in the components and we keep all the backend points of contact in one structured place. 

Please take a look on `state/api/contests` [as an example](https://github.com/hicommonwealth/commonwealth/blob/6ff154894b84fedaabff9bbb78a247fda147cc93/packages/commonwealth/client/scripts/state/api/contests/getContests.ts#L8). It uses trpc but we still create files and hooks in the way we did in the past.  

So please create state/api/subscriptions directory and inside
- deleteThreadSubscription.ts
- getThreadSubscriptions.ts
and inside create a hook that wraps trpc. Please follow the pattern